Roslyn.sln

```diff
@@ -330,6 +330,12 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CSharpCompilerTestUtilities
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DeployDesktopTestRuntime", "src\Test\DeployDesktopTestRuntime\DeployDesktopTestRuntime.csproj", "{23683607-168A-4189-955E-908F0E80E60D}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VisualStudioTestSetup", "src\VisualStudio\TestSetup\VisualStudioTestSetup.csproj", "{A88AB44F-7F9D-43F6-A127-83BB65E5A7E2}"
```

Comment:
Is this a replacement or move of the internal project we already have with this same name?